front/src/Api/Events/AskPositionEvent.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+
+export const isAskPositionEvent = z.object({
```

Comment:
@cesar-cardinale this file is used in `front/src` or already moved in `front/chat/src`?